src/main/scripts/jbang

```diff
@@ -127,21 +127,15 @@ if [ -f "$abs_jbang_dir/jbang.jar" ]; then
 elif [ -f "$abs_jbang_dir/.jbang/jbang.jar" ]; then
   jarPath=$abs_jbang_dir/.jbang/jbang.jar
 else
-  if [[ ! -f "$JBDIR/bin/jbang.jar" || ! -f "$JBDIR/bin/jbang" ]]; then
-    echo "Downloading JBang..." 1>&2
-    mkdir -p "$TDIR/urls"
-    jburl="https://github.com/jbangdev/jbang/releases/latest/download/jbang.tar"
-    download $jburl "$TDIR/urls/jbang.tar"
-    if [ $retval -ne 0 ]; then echo "Error downloading JBang" 1>&2; exit $retval; fi
-    echo "Installing JBang..." 1>&2
-    rm -rf "$TDIR/urls/jbang"
-    tar xf "$TDIR/urls/jbang.tar" -C "$TDIR/urls"
-    if [ $retval -ne 0 ]; then echo "Error installing JBang" 1>&2; exit $retval; fi
-    mkdir -p "$JBDIR/bin"
-    rm -f "$JBDIR/bin/jbang" "$JBDIR/bin"/jbang.*
-    cp -f "$TDIR/urls/jbang/bin"/* "$JBDIR/bin"
-  fi
-  "$JBDIR"/bin/jbang "$@"
+  zidir=$(mktemp -d "$(basename $0).XXXXXXXXXX")
```

Comment:
whats the semantics for mktemp created directories? deleted at boot or ?

Reply:
No, they get deleted either by the `rm` on line 138 (if execution is all green) or by the `trap` statement on line 131 if anything goes wrong or the user cancels the job.

Reply:
And they get created in /tmp so if for some reason they still manage to evade any of that cleanup they will follow whatever cleanup is defined for your OS.

Reply:
ack